packages/graphql/src/stitching/remoteSchemaWrapped.ts

```diff
@@ -65,13 +68,16 @@ export const remoteSchemaWrapped = wrapSchema<DataContext>({
       // When we respond eagerly with a result, but receive an updated value
       // for the query, we can "push" the data down using the pushFragment subscription
       onUpdatedResult (result) {
-        obj.context?.graphql.pushResult({
```

Comment:
Looks like the TS typings are good with it, but the original code treated `context` as nullable. Any risk of that here?

Reply:
No, because it's assigned above so it can't change